tools/cockpit.spec

```diff
@@ -71,7 +71,10 @@ BuildRequires: pcp-libs-devel
 BuildRequires: gdb
 
 %if %{defined gitcommit}
+# on fedora 24, nodejs provides npm
+%if 0%{?fedora} > 0 && 0%{?fedora} < 24
```

Comment:
> What about rhel and centos? How is npm packaged there?

I'm not sure, but the fedora-24 build failed without the change. The others still work, so I didn't change them.